src/main/scala/org/scalajs/tools/tsimporter/Importer.scala

```diff
@@ -268,6 +268,9 @@ class Importer(val output: java.io.PrintWriter) {
           case TypeName("Array") => QualifiedName.Array
           case TypeName("Function") => QualifiedName.FunctionBase
           case TypeName("object") => QualifiedName.Object
+          case TypeName("PromiseLike") => QualifiedName.Thenable
+          case TypeName("ReadonlyArray") => QualifiedName.JsArray
```

Comment:
`ReadonlyArray` is more complicated than that, because `ReadonlyArray[T]` should translate to `js.Array[_ <: T]`, not `js.Array[T]`.

Reply:
You'll probably have to enhance `sc.TypeRef` to support that structure. Perhaps by changing the `List[TypeRef]` into a `List[TypeRefOrWildcard]` with `case class Wildcard(upperBound: Option[TypeRefOrWildcard])`

Reply:
OK, Ive implemented that, please review